jupyterlab_templates/extension.py

```diff
@@ -2,15 +2,22 @@
 import json
 import os
 import os.path
-from io import open
-
 import jupyter_core.paths
 
-from notebook.base.handlers import IPythonHandler
+from io import open
+from datetime import datetime
 from notebook.utils import url_path_join
+from notebook.base.handlers import IPythonHandler
 
+DATE_FORMAT = "%Y-%m-%d"
+TIME_FORMAT = "%H:%M:%S"
 DEFAULT_USERNAME = "anonymous"
 USERNAME_TEMPLATE = "##username##"
+TEMPLATES_TO_FUNCTION = {
```

Comment:
Not really pretty that username is calculated in a different way, but because there are no other use-cases at the moment I guess its not necessary to make it generic..

Reply:
i agree yet i'm afraid that for someone new to this code, in the simple usecase it could be easier to add something like that. there for i did it like that anyway